subscription_oca/README.rst

```diff
@@ -87,6 +87,10 @@ Contributors
 
   * Ilyas <irazor147@gmail.com>
 
+* `Komit <https://komit-consulting.com>`__:
```

Comment:
Hi, no need to add this here as you already added it in the contributor section

Reply:
This change is auto-generated by pre-commit.